### PR TITLE
Add Farbwerk 360 support to aquacomputer driver

### PR DESCRIPTION
**Pull request by aleksamagicka**
Following up on #482, this PR adds support and tests for reading temperature sensors of the Aquacomputer Farbwerk 360 RGB controller.

Related: #438 

---

Checklist:

- [x] Adhere to the [development process]
- [x] Conform to the [style guide]
- [x] Verify that the changes work as expected on real hardware
- [x] Add automated tests cases
- [x] Verify that all (other) automated tests (still) pass
- [x] Update the README and other applicable documentation pages
- [x] Update the `liquidctl.8` Linux/Unix/Mac OS man page
- [x] Update or add applicable `docs/*guide.md` device guides
- [ ] Submit relevant data, scripts or dissectors to https://github.com/liquidctl/collected-device-data

New CLI flag?

- [ ] Adjust the completion scripts in `extra/completions/`

New device?

- [x] Regenerate `extra/linux/71-liquidctl.rules` (instructions in the file header)
- [x] Add entry to the README's supported device list with applicable notes (at least `en`)

New driver?

- [x] Document the protocol in `docs/developer/protocol/`

[development process]: https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md
[style guide]: https://github.com/liquidctl/liquidctl/blob/main/docs/developer/style-guide.md

### CODE REVIEW

File: liquidctl/driver/aquacomputer.py [hunk 1]
```diff
@@ -41,9 +48,13 @@
 
 
 class Aquacomputer(UsbHidDriver):
-    # Support for hwmon: aquacomputer_d5next, sensors - 5.15+
+    """Support for hwmon: aquacomputer_d5next driver
+    - D5 Next watercooling pymp: sensors - 5.15+
+    - Farbwerk 360: sensors - 5.18+
+    """
```
Comment (jonasmalacofilho) [inline liquidctl/driver/aquacomputer.py:54]: I think it's better to move this note to the module docstring, or to a simple comment.  It's currently hard for non-Linux users to parse, and as the entire class docstring, it also suggests that this driver is only about hwmon support.

File: tests/test_aquacomputer.py [hunk 2]
```diff
@@ -130,9 +138,105 @@ def test_d5next_set_fixed_speeds_not_supported(mockD5NextDevice):
         mockD5NextDevice.set_fixed_speed("pump", 84)
 
 
-def test_kraken_speed_profiles_not_supported(mockD5NextDevice):
```
Comment (jonasmalacofilho) [inline tests/test_aquacomputer.py:133]: Oh, I totally missed this before, ops : )

File: liquidctl/driver/aquacomputer.py (hunk 1 above)
Comment (aleksamagicka) [reply liquidctl/driver/aquacomputer.py]: Thanks, moved it to the module docstring.

**Comment by jonasmalacofilho** [comment]
Merged, thanks!

**Pull request merged**